cluster-up/cluster/k8s-provider-common.sh

```diff
@@ -57,14 +57,25 @@ function deploy_cnao() {
         $kubectl create -f /opt/cnao/operator.yaml
 
         if [ "$KUBVIRT_WITH_CNAO_SKIP_CONFIG" != "true" ]; then
-            $kubectl create -f /opt/cnao/network-addons-config-example.cr.yaml
+            create_network_addons_config
         fi
 
         # Install whereabouts on CNAO lanes
         $kubectl create -f /opt/whereabouts
     fi
 }
 
+function create_network_addons_config() {
+    local nac="/opt/cnao/network-addons-config-example.cr.yaml"
+    if [ "$KUBEVIRT_WITH_MULTUS_V3" == "true" ]; then
```

Comment:
I think that if both `KUBVIRT_WITH_CNAO_SKIP_CONFIG` and `KUBEVIRT_WITH_MULTUS_V3` are defined
it will still deploy the CR even that `KUBVIRT_WITH_CNAO_SKIP_CONFIG` says do not, right ?

Reply:
No, because `create_network_addons_config()` is called when `KUBVIRT_WITH_CNAO_SKIP_CONFIG` is **not** true.

Reply:
Right
so if one wants to use `KUBVIRT_WITH_CNAO_SKIP_CONFIG` but to deploy V3
he would need to do it manually or there is a way still ?